roles/kubernetes-apps/ingress_controller/cert_manager/templates/cert-manager.yml.j2

```diff
@@ -947,6 +947,9 @@ spec:
           - --v=2
           - --cluster-resource-namespace=$(POD_NAMESPACE)
           - --leader-election-namespace={{ cert_manager_leader_election_namespace }}
+{% for extra_arg in cert_manager_controller_extra_args | default([]) %}
```

Comment:
Hi quick nits, could you rather default this new var to empty array in `roles/kubernetes-apps/ingress_controller/cert_manager/defaults/main.yml`, apart from that lgtm. Thanks!